python/telemetry/nila.py

```diff
@@ -0,0 +1,237 @@
+##################################################################
+#     This Decoder File Belongs to HEX20 Labs India Pvt Ltd      #
+#     Decoder for NILA 3U Satellite                              #
+##################################################################
+
+from .ax25 import Header
+from ..adapters import PolynomialAdapter
+from construct import Adapter, BitStruct, BitsInteger, Enum, Flag, Float32l, \
+    If, Int16sl, Int16ul, Int32sl, Int32ul, Int8sl, \
+    Int8ub, Int8ul, Padding, Struct, Switch
+
+pannel_volt_poly = [0.000000, 0.001]
+pannel_current_poly = [0.000000, 0.0005]
+
+subsystem_current_poly = [0.000000, 0.0005]
+subsystem_volt_poly = [0.000000, 0.001]
+
+aux_current_poly = [0.000000, 0.0005]
+aux_volt_poly = [0.000000, 0.001]
+
+solar_panel_temp_poly = [91.394, -0.08949, 3.55e-05, -6.26e-09, 1.89e-13]
+batt_thermistor_temp_poly = [
+    87.1751343, -0.0786252941, 0.0000272861362, -0.00000000402689014]
+subsystem_temp_poly = [91.394, -0.08949, 3.55e-05, -6.26e-09, 1.89e-13]
+
+nila_header = Struct(
+    'sync_word_1' / Int8ul,
+    'sync_word_2' / Int8ul,
+    'sync_word_3' / Int8ul,
+    'version' / Int8ul,
+    'ap_id' / Int8ul,
+    'seq_no' / Int16ul,
+    'pkt_len' / Int16ul,
```

Comment:
Why is there a packet length field? AX.25 frames are delimited by HDLC, so an explicit packet length field is not needed.